src/jwt.interceptor.ts

```diff
@@ -40,15 +40,15 @@ export class JwtInterceptor implements HttpInterceptor {
 
   isWhitelistedDomain(request: HttpRequest<any>): boolean {
     const requestUrl: any = parse(request.url, false, true);
-
+    const requestHost: string = requestUrl.host || (location && location.host)
```

Comment:
In order to not block potential universal support, can we do a `typeof` check here on location?

Reply:
![image](https://user-images.githubusercontent.com/1164534/58976512-8b795a00-87bf-11e9-9847-0f4f94c690ae.png)

```
typeof location === 'object'
```

???

Reply:
Done